crates/viewer/re_viewer/src/web.rs

```diff
@@ -372,10 +382,12 @@ fn create_app(
         persist_state: app_options.persist.unwrap_or(true),
         is_in_notebook: app_options.notebook.unwrap_or(false),
         expect_data_soon: None,
-        force_wgpu_backend: None,
+        force_wgpu_backend: app_options.render_backend.clone(),
```

Comment:
It looks like we could avoid this bug (and future bugs of the same type) by destructing `AppOptions` instead, at the top of the function:

```rs
let AppOptions { persist, notebook, render_backend, … } = app_options
```

```suggestion
        force_wgpu_backend,
```